### PR TITLE
Standardized Readme

### PR DESCRIPTION
**Pull request by RichardLitt**
See https://github.com/ipfs/community/issues/124

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
->
-> We will be writing a set of docs, posts, tutorials, and talks to explain what p2p is, why it is tremendously useful, and how it can help your existing and new projects. But in the meantime, check out
->
-> - [**The IPFS Network Spec**](https://github.com/ipfs/specs/tree/master/libp2p), which grew into libp2p
-> - [**go-libp2p implementation**](https://github.com/ipfs/go-libp2p)
-> - [**js-libp2p implementation**](https://github.com/diasdavid/js-libp2p)
 
-# Contribute
+libp2p is the product of a long and arduous quest of understanding -- a deep dive into the internet's network stack, and plentiful peer-to-peer protocols from the past. Building large scale peer-to-peer systems has been complex and difficult in the last 15 years, and libp2p is a way to fix that. It is a "network stack" -- a protocol suite -- that cleanly separates concerns, and enables sophisticated applications to only use the protocols they absolutely need, without giving up interoperability and upgradeability. libp2p grew out of IPFS, but it is built so that lots of people can use it, for lots of different projects.
```
Comment (whyrusleeping) [inline README.md]: why did you put this all into one line?

Comment (RichardLitt) [inline README.md]: It was already in one line?

**Comment by RichardLitt** [comment]
@whyrusleeping Word on this?

File: README.md [hunk 2]
```diff
@@ -41,24 +40,26 @@ $ cd $GOPATH/src/github.com/ipfs/go-libp2p
...
 ```
 
-# Examples
+## Usage
+
+`go-libp2p` repo will be a place holder for the list of Go modules that compose Go libp2p, as well as its entry point.
+
+## Examples
 
 Examples can be found on the [libp2p-examples repo](https://github.com/whyrusleeping/libp2p-examples/)
```
Comment (hsanjuan) [inline README.md]: See #133 as well

Comment (RichardLitt) [inline README.md]: This can be merged automatically, with #133. Any comments on this PR? @hsanjuan how does it look, otherwise?

Comment (hsanjuan) [inline README.md]: LGTM, I'm already a fan of your standarized readmes :)

File: README.md [hunk 3]
```diff
@@ -1,37 +1,36 @@
-[libp2p](https://github.com/libp2p/specs) implementation in Go.
-===================
+# go-libp2p
+
+![](https://raw.githubusercontent.com/diasdavid/specs/libp2p-spec/protocol/network/figs/logo.png)
```
Comment (lgierth) [inline README.md]: The logo should be moved into this repo here

File: README.md [hunk 4]
```diff
@@ -1,37 +1,36 @@
...
-libp2p implementation in Go is a work in progress. As such, there's a few things you can do right now to help out:
- - Go through the modules below and **check out existing issues**. This would be especially useful for modules in active development. Some knowledge of IPFS/libp2p may be required, as well as the infrasture behind it - for instance, you may need to read up on p2p and more complex operations like muxing to be able to help technically.
- - **Perform code reviews**.
- - **Add tests**. There can never be enough tests.
+We will be writing a set of docs, posts, tutorials, and talks to explain what p2p is, why it is tremendously useful, and how it can help your existing and new projects. But in the meantime, check out:
 
-# Usage
+- [**The IPFS Network Spec**](https://github.com/ipfs/specs/tree/master/libp2p), which grew into libp2p
+- [**go-libp2p implementation**](https://github.com/ipfs/go-libp2p)
+- [**js-libp2p implementation**](https://github.com/diasdavid/js-libp2p)
```
Comment (lgierth) [inline README.md]: needs to point to libp2p/specs and libp2p/js-libp2p respectively

**Comment by RichardLitt** [comment]
@lgierth fixed.

File: README.md [hunk 5]
```diff
@@ -109,3 +110,24 @@ We want to maintain history, so we'll use git-subtree for extracting packages.
...
 ```
+
+## Maintainers
+
+Captain: [@whyrusleeping](https://github.com/whyrusleeping)
+
+## Contribute
+
+libp2p implementation in Go is a work in progress. As such, there's a few things you can do right now to help out:
+ - Go through the modules below and [**check out existing issues**](https://github.com/libp2p/go-libp2p/issues). This would be especially useful for modules in active development. Some knowledge of IPFS/libp2p may be required, as well as the infrasture behind it - for instance, you may need to read up on p2p and more complex operations like muxing to be able to help technically.
```
Comment (lgierth) [inline README.md]: should either remove the go-libp2p reference or also mention js-libp2p

Comment (RichardLitt) [inline README.md]: It should only mention go!

**Comment by RichardLitt** [comment]
@lgierth This should be good to go, no?

**Comment by RichardLitt** [comment]
@whyrusleeping Apparently you're the merge master for this one.

**Comment by diasdavid** [comment]
I believe this one can be closed. The README got a summer cleaning today :)

**Pull request closed without merging**